### ISSUE TITLE
[TTS Extended] - Unavailable on Android Market

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#46]
```
1. I've downloded Google Translator for android by QR code at 
http://translate.google.ru/about/intl/ru_ALL/tour.html#mobile
2. After start up I was requested to install SVOX data. It was installed 
successfuly.
3. In the same messege there was request to install "Расширенный 
синтезатор речи" (Expanded speech sequencer). I pressed buton 
"Install". But error message has appeared "No results in Android Market for 
pname:com.google.tts query"
4. By usual search I found this project home page. I've downloaded  
tts_3.1_market.apk. And installed it from PC.

I'm using HTC Wildfire, Android 2.1, WiFi connection at home, Moscow, Russia.

Why I can't get TTS from Android Market?
```

Original issue reported on code.google.com by `artyom.r...@gmail.com` on 7 Sep 2010 at 7:44

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem with HTC EVO 4G, that came with Froyo. The applicaton 
does work, once installed from the .apk
```

Original comment by `leonelga...@gmail.com` on 7 Sep 2010 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
I suspect this is matter filter based on location as same issue for me in UK.
ie, Update to TTS was reported by Appbrain. However, not found on Market. Nexus 
One, FRF91 (stock), T--Mobile,  UK.
```

Original comment by `ealingba...@gmail.com` on 11 Sep 2010 at 7:47

**Comment by GoogleCodeExporter** [comment]
```
TTS Extended is available on Android Market for devices running 1.5 to 2.1.  
Support for TTS Extended has been capped at 2.1 due to platform TTS being added 
in Froyo.  I will investigate to see if locale filters are in effect.
```

Original comment by `caseybur...@google.com` on 2 Jun 2011 at 11:19
- Changed title: **[TTS Extended] - Unavailable on Android Market**
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
There are no locale or device filters in effect for TTS Extended on Android 
Market.  If you have a device running 1.5 to 2.1, it will be available for 
download.
```

Original comment by `caseybur...@google.com` on 3 Jun 2011 at 6:00
- Changed state: **Invalid**
